### DISCUSSION TITLE
Can capacitor be used inside cordova project

### DISCUSSION THREAD

**Discussion by debashish4** [ionic-team/capacitor#4534] [Q&A] [answered]
I have a cordova project and I would like to use he capacitor plugins inside Cordova project because some of the Cordova plugins is not working as it should but capacitor has those plugins and I would like to use those plugins inside my Cordova project.

**Comment by jcesarmobile** [comment, marked as answer]
No, it's not possible, Capacitor has some code to add Cordova plugin compatibility, but Cordova doesn't have any code for Capacitor plugin compatibility
